lib/plot/falco_plot_progress_dst.m

```diff
@@ -200,7 +200,7 @@
     % - Throughput plots 
     % - 
 
-end
+end % if plot
```

Comment:
falco_plot_progress_dst.m is a DST-specific function and therefore needs to stay the same in order to not impact DST operation.